hca_util/aws_client.py

```diff
@@ -29,8 +29,11 @@ def is_valid_credentials(self):
             if arn.endswith(f'user/{IAM_USER_CONTRIBUTOR}'):
                 self.is_contributor = True
             return True
-        except:
-            return False
+        except Exception as e:
+            if e is not KeyboardInterrupt:
+                return False
+            else:
+                raise e
```

Comment:
the caller is probably not handling this exception. please check.

Reply:
In both cases the ultimate caller is __main__.py which catches the KeyboardInterrupt to abort the entire process.